libs/components/src/lib/card/base/index.tsx

```diff
@@ -92,7 +92,9 @@ export const BaseCard: React.FC<BaseCardProps> = ({
               'flex flex-col',
               textColorVariantClass[color],
               sizeVariantTextGap[size],
+              // 'gap-gap-lg',
```

Comment:
can remove this comment if it's not needed.

Reply:
Done with removing the commented line